ckanext/dataset_subscriptions/plugin.py

```diff
@@ -1,14 +1,91 @@
 import ckan.plugins as plugins
 import ckan.plugins.toolkit as toolkit
+import ckan.logic as logic
+import ckan.lib.helpers as helpers
+import ckan.lib.email_notifications as email_notifications
+import ckan.model as model
+import ckan.lib.base as base
+from ckan.common import ungettext, config
```

Comment:
you probably should use `ckan.toolkit._` for translations. Is there some specific reason on using `ungettext`?

Reply:
I had to use ckan.toolkit.ungettext because of singular/plural choice.